packages/drivers/odsp-driver-definitions/src/errors.ts

```diff
@@ -52,11 +52,12 @@ export enum OdspErrorType {
  * Base interface for all errors and warnings
  * Superset of IDriverErrorBase, but with Odsp-specific errorType
  */
-export interface IOdspError {
+export interface IOdspError extends Omit<IDriverErrorBase, "errorType"> {
```

Comment:
why not IAnyDriverError  here?

Reply:
Probably dependencies.  `IAnyDriverError` is in `driver-utils` at the moment, mostly for pragmatic reasons.  Would you agree with moving it to driver-definitions?

Reply:
i'd rather not add anything to definitions with the current approach, as that is just more to deprecate when we move to error api

Reply:
Sounds good to me